cli_tools/import_precheck/check_disks_linux.go

```diff
@@ -60,47 +87,114 @@ func (c *disksCheck) getName() string {
 	return "Disks Check"
 }
 
-func (c *disksCheck) run() (*report, error) {
-	r := &report{name: c.getName()}
-	var out string
-	var err error
-	if c.lsblkOverride != nil {
-		out, err = c.lsblkOverride()
-	} else {
-		out, err = c.lsblk()
+// mountPoints supports listing the mounts on a system, and determining which block device(s)
+// the mount is physically contained. This is helpful since technologies such as LVM allow
+// logical mounts to span physical devices.
+type mountPoints struct {
+	mounts []struct {
+		dir       string
+		hierarchy []string
```

Comment:
is `hierarchy` a list of block devices? Or a path split into elements? It's not clear what this field represents

Reply:
good question -- described with a comment (it's a list of block devices. the first is the physical device, then partition, then virtual devices)